gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -712,7 +712,8 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
           "(true/false):(comma separated ACCELERATOR(s)). "
           "true/false determines if accelerator is to be used. "
           "list of accelerators determines the backend (ignored with false). "
-          "Example, if GPU, NPU can be used but not CPU - true:(GPU,NPU,!CPU). "
+          "Example, if GPU, NPU can be used but not CPU - true:npu,gpu,!cpu."
+          "The full list of accelerators can be found in nnstreamer_plugin_api_filter.h."
```

Comment:
Please add a space between ```.``` and ```"``` at the end of line. I.e., ```."``` ==> ```. "```

Reply:
Fixed, thanks for pointing it out.